### DISCUSSION TITLE
[Q] Zigbee Protocol Version of Devices

### DISCUSSION THREAD

**Discussion by Sineos** [Koenkk/zigbee2mqtt#18886] [Devices]
Is there a way how to determine what protocol version, e.g. 1.2 or 3.0, a device uses? Ideally, a complete list of all connected devices.

**Comment by Koenkk** [comment]
AFAIK not

**Reply by Sineos** [reply]
That's kind of disappointing. Thanks for your insight, nevertheless. Appreciated.

**Comment by RobLeighton67** [comment]
Does the ZCL version offer any clue ?
